### ISSUE TITLE
[Readwise] Opening 

### ISSUE THREAD

**Issue by aaronjz** [raycast/extensions#8860]
### Extension

https://www.raycast.com/natterstefan/readwise

#### Error:
```
The API file appears to be corrupted. You may need to reinstall Raycast.


```

### Steps To Reproduce

Run Open Daily Review... See error.

### Current Behaviour

When opening Daily Review, the error above shows. This only happened after Raycast updated itself to 1.60.0. It was working fine before that.

### Expected Behaviour

To open the Daily Review.

**Comment by sxn** [comment]
hey, thanks for the report. This is an issue with Raycast, not the extension. We're currently addressing it and will deploy a hotfix shortly. Sorry for the hassle 🙏

**Comment by sxn** [comment]
We've just published a hotfix (`1.60.1`) that should address the issue. You can download it from https://www.raycast.com, or run the `Check For Updates` command. ❤️

**Comment by aaronjz** [comment]
Thank you. Things are back to normal! Grateful for the quick fix.

Aaron
